src/main/java/uk/ac/ebi/ena/readtools/loader/common/consumer/DataConsumable.java

```diff
@@ -0,0 +1,4 @@
+package uk.ac.ebi.ena.readtools.loader.common.consumer;
+
+public interface DataConsumable {

```

Comment:
Rename IluminaSpot to FastqSpot please. Remove DataConsumable interface and replace with FastSpot.

Reply:
Are suggesting this interface be renamed to 'FastSpot' or should it be removed instead?

Reply:
DataConsumable interface to be removed and wherever this interface is used currently we should use FastqSpot instead.

Reply:
This is the marker interface that I added after you suggestion on limiting the T1 and T2 generic types. FastqSpot is a class. Along with DataSpot class, these two are used by the producers and consumers to read and write fastq data.

Please see the DataConsumer.java file to see how this interface is being used.

Reply:
OK, got it and spend a bit more time reading the code. This is actually linked to the removal of the @ProducibleData @ProducibleDataChecker annotations. If we replace these with an interface then this interface should be common to FastqSpot and DataSpot and this is currently 'DataConsumable'. For clarity I suggest we rename 'DataConsumable' to 'Spot' and add the methods from @ProducibleData @ProducibleDataChecker into 'Spot'. Then we can trivially see what is common between FastqSpot and DataSpot (i.e. what is 'Spot'). It is not clear (yet) why we need two different classes FastqSpot and DataSpot instead of just one.